repos/system_upgrade/el7toel8/actors/authselectscanner/libraries/library.py

```diff
@@ -0,0 +1,273 @@
+import ConfigParser
+import errno
+import re
+import StringIO
+import textwrap
+
+from leapp.libraries.common.pam import PAM
+from leapp.libraries.stdlib import run, CalledProcessError 
+from leapp.models import Authselect
+
+
+def read_file(file):
+    """
+        Read file contents. Return empty string if the file does not exist.
+    """
+    try:
+        with open(file, "r") as f:
+            return f.read()
+    except IOError as err:
+        if err.errno == errno.ENOENT:
+            return ""
+        else:
+            raise 
+
+
+def is_service_enabled(service):
+    """
+        Return true if @service is enabled with systemd, false otherwise.
+    """
+    try:
+        run(["/usr/bin/systemctl", "is-enabled", service + ".service"])
```

Comment:
Please use string formatting instead of concatenation `{}.service.format(service)`